backend/asmgen.mli

```diff
@@ -39,6 +39,27 @@ val compile_implementation
   -> Lambda.program
   -> unit
 
+(** [compile_implementation_linear] reads Linear IR from [progname] file
+    produced by previous compilation stages (for example,
+    using "ocamlopt -save-ir-after" and "ocamlfdo opt")
+    and continues compilatin from [Emit].
+
+    Correctness: carefully consider any use of [Config], [Clflags],
```

Comment:
I think there should be another comment at the top of both emitters pointing at this one.